EIPS/eip-1753.md

```diff
@@ -0,0 +1,249 @@
+---
+eip: 1753
+title: Smart Contract Interface for Licences
+author: Lucas Cullen (@BitcoinBrisbane), Kai Yeung (@CivicKai), Anna Crowley <annaelizabethcrowley@gmail.com>, Caroline Marshall <caroline.marshall888@gmail.com>, Katrina Donaghy, <katrina@civicledger.com>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-02-06
+---
+
+# Abstract
```

Comment:
This must be indented 2 levels (`## Abstract`).

Reply:
Fixed.